circuit/program/src/data/record/to_bits.rs

```diff
@@ -25,7 +25,7 @@ impl<A: Aleo> ToBits for Record<A, Plaintext<A>> {
         let data_bits_le = self
             .data
             .iter()
-            .flat_map(|(identifier, entry)| vec![identifier.to_bits_le(), entry.to_bits_le()])
+            .flat_map(|(identifier, entry)| [identifier.to_bits_le(), entry.to_bits_le()])
             .flatten()
```

Comment:
hard to say without benchmarks but wouldn't not introducing an array be faster?
```suggestion
            .flat_map(|(identifier, entry)| identifier.to_bits_le().chain(entry.to_bits_le()))
            .flatten()
```

Reply:
it's unlikely that there would be a difference in performance, but there is a more elegant way to do this; I'll add a new commit